src/Facades/ScoutApm.php

```diff
@@ -1,12 +1,31 @@
 <?php
+declare(strict_types=1);
 
 namespace Scoutapm\Laravel\Facades;
 
+use Closure;
 use Illuminate\Support\Facades\Facade;
+use Scoutapm\Events\Span\Span;
 
+/**
+ * @method static void connect()
```

Comment:
I slipped these in to help consumers with static analysis when using Facades. Without these, most IDEs will complain that methods that don't exist are being called due to the hidden magic being used to make Facades work.

Reply:
👍